returnn/tf/util/data.py

```diff
@@ -2928,7 +2928,7 @@ def copy_template_unpack_batch(self):
             dim_tags.append(virtual_dim.dim_tag)
             batch = batch.copy_remove_dim(virtual_dim)
           elif not new_batch_dim_tag:
-            new_batch_dim_tag = DimensionTag(kind=DimensionTag.Types.Batch)
+            new_batch_dim_tag = DimensionTag(kind=DimensionTag.Types.Batch, description="unpack_batch_dim")
             dim_tags.append(new_batch_dim_tag)
```

Comment:
Wait for #822. And then just do:

```suggestion
            dim_tags.append(batch_dim)
```

Reply:
Alright, thanks :+1:

Reply:
#822 is merged now. Can you rebase and do this change?

Reply:
Yes, done. I just need to keep track of the `new_batch_dim_tag`.